src/screens/Profile/DebugInfo/index.tsx

```diff
@@ -206,6 +288,52 @@ function mapEntry(key: string, value: any) {
   }
 }
 
+function LabeledSlider({
+  label,
+  min = 0,
+  max,
+  step = 0.1,
+  defaultValue,
+  initialValue,
+  onSetValue,
+}: {
+  label: string;
+  min?: number;
+  max: number;
+  step?: number;
+  defaultValue?: number;
+  initialValue?: number;
+  onSetValue: (n: number) => void;
+}): JSX.Element {
+  const [pref, setPref] = useState(initialValue || defaultValue);
+
+  return (
+    <Sections.GenericItem>
+      <ThemeText
+        onPress={
+          defaultValue
+            ? () => {
+                onSetValue(defaultValue);
+                setPref(defaultValue);
+              }
+            : undefined
+        }
+      >
+        {label}: {pref?.toFixed(1)}
+      </ThemeText>
+      <Slider
+        style={{width: '100%'}}
+        minimumValue={min}
+        maximumValue={max}
+        step={step}
+        value={pref}
+        onValueChange={(n) => setPref(n)}
```

Comment:
Litt sånn nitpicky og bare for enkelhetsskyld her. Her er det egentlig unødvendig å generere en egen funksjon for hver gang, så lenge det bare er å sende verdien videre. Så her kunne man like gjerne gjøre:
```suggestion
        onValueChange={setPref}
```